packages/trace-viewer/src/ui/callTab.css

```diff
@@ -59,7 +59,6 @@
 
 .copy-icon {
   display: none;
-  margin-right: 4px;
```

Comment:
While we are here, let's change the selector to `.call-line .copy-icon`. Otherwise, it currently affects not only the call tab, but any other copy icon in the trace viewer.

Additionally, let's add some `margin-left` to space the copy icon from the value a bit.